source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst

```diff
@@ -114,34 +116,60 @@ Event Order and Concurrency
 ---------------------------
 
 Events MUST be published in the order they were applied in the driver.
-Events MUST NOT be published concurrently for the same topology id or server id, but MAY be published concurrently for differing topology ids and server ids.
+
+Events MUST NOT be published concurrently for the same topology id or server
+id, but MAY be published concurrently for differing topology ids and server ids.
 
 Heartbeats
 ----------
 
-The driver MUST guarantee that every ServerHeartbeatStartedEvent has either a correlating ServerHeartbeatSucceededEvent or ServerHeartbeatFailedEvent.
+The driver MUST guarantee that every ServerHeartbeatStartedEvent has either
+a correlating ServerHeartbeatSucceededEvent or ServerHeartbeatFailedEvent.
 
 Error Handling
 --------------
 
-If an exception occurs while sending the ismaster operation to the server, the driver MUST generate a ServerHeartbeatFailedEvent with the exception or message and re-raise the exception. The SDAM mandated retry of the ismaster call should be visible to consumers.
+If an exception occurs while sending the ismaster operation to the server,
+the driver MUST generate a ServerHeartbeatFailedEvent with the exception or
+message and re-raise the exception. The SDAM mandated retry of the ismaster
+call should be visible to consumers.
 
 Topology Ids
 ------------
 
-These MUST be a unique value that is specific to the Topology in which the events are fired. The language may decide how to generate the value and what type the value is, as long as it is unique to the Topology. The id MUST be created once when the Topology is created and remain the same until the Topology is destroyed.
+These MUST be a unique value that is specific to the Topology in which the
+events are fired. The language may decide how to generate the value and what
+type the value is, as long as it is unique to the Topology. The id MUST be
+created once when the Topology is created and remain the same until the
+Topology is destroyed.
 
 Topology Description
 --------------------
 
-The TopologyDescription object MUST expose the new methods defined in the API below, in order for subscribers to take action on certain conditions based on the driver options.
+The TopologyDescription object MUST expose the new methods defined in the
+API below, in order for subscribers to take action on certain conditions
+based on the driver options.
 
 TopologyDescription objects MAY have additional methods and properties.
 
 Initial Server Description
 --------------------------
 
-ServerDescriptions MUST be initialized with a default description in an “unknown” state, guaranteeing that the previous description in the events will never be null.
+ServerDescriptions MUST be initialized with a default description in an
+"unknown" state, guaranteeing that the previous description in the events will
+never be null.
+
+Meaningful Changes
+------------------
+
+If the driver processes a ServerDescription which is equal (as server
```

Comment:
I find this paragraph incredibly difficult to parse, and I'm not sure what this prose buys us beyond the existing definition of `ServerDescription` equality in the top-level SDAM spec. Is any new information introduced here?

Reply:
This change is no longer part of this PR.